src/js/components/account/user-registration.js

```diff
@@ -116,6 +117,21 @@ export class UserRegistration {
       $form.append($formElementsContainer);
     });
 
+    const $privacyContainer = $("<div />");
+    const $privacyPolicyLabel = getLabel($webflowForm, {
+      htmlFor: "privacy-policy-checkbox",
+      text: "Privacy policy label",
+    });
+
+    const $privacyPolicyInput = getCustomCheckbox({
+      identifier: "privacy-policy-checkbox",
+      name: "Privacy policy checkbox",
```

Comment:
Not a train smash but, for consistency perhaps:

```suggestion
      name: "privacy-policy-checkbox",
```